test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -547,26 +526,6 @@ private static CompositeModelBinder CreateCompositeBinder(IModelBinder mockIntBi
             return shimBinder;
         }
 
-        private static IModelValidatorProvider GetValidatorProvider(params IModelValidator[] validators)
```

Comment:
When validation used to take place *during* model binding, this test made sense. It was just never cleaned up.